### DISCUSSION TITLE
Always tell the player that he has op

### DISCUSSION THREAD

**Discussion by masmc05** [PurpurMC/Purpur#877] [__Completed__] [answered]
All the features are like f3+f4 are just sent commands, but if i would like to disable op on the server (like with luck perms), the client says that you don't have permission (even if i have, but not op), and plugins like f3nperm are buggy

**Comment by masmc05** [comment]
oh, forgot to mention to make that as a config option, but i think it's obvious

**Comment by BillyGalbreath** [comment, marked as answer]
you dont need f3nperm plugin. purpur has something better built in

https://purpurmc.org/docs/Permissions/#purpurdebugf3n

just give that perm and purpur will tell the client you're op, even though you're not.

**Comment by masmc05** [comment]
oh, sorry seems like i didn't see that, thanks
